src/questions/css-questions.md

```diff
@@ -38,6 +38,39 @@ permalink: /questions/css-questions/index.html
 * How is clearfix css property useful?
 * Can you explain the difference between px, em and rem as they relate to font sizing?
 * Can you give an example of a pseudo class? Can you provide an example use case for a pseudo class? 
+  * What new pseudo-classes have appeared in the last year or two?
```

Comment:
Not a huge fan of these kind of questions. There's not much to get out of the answer. Please drop this one.

Reply:
are you sure?
when I ask people about it, I sometimes get an answer like “I use the tried and true old stuff rather than something new”, which for me is a huge minus